### DISCUSSION TITLE
What is the source of the data?

### DISCUSSION THREAD

**Discussion by l3onh4rd** [f1db/f1db#29] [Q&A] [answered]
I just found this [question](https://github.com/f1db/f1db/discussions/6) while I was wondering about the source of this amazing data repository. Am I right that you enter all data manually? @marceloverdijk

**Comment by marceloverdijk** [comment, marked as answer]
Hi @l3onh4rd , I collected the data over the past years from various sources, see also this answer: https://github.com/f1db/f1db/discussions/6#discussioncomment-4213146

After every race I indeed create new entries for the results.
